### PR TITLE
Add max_bytes parameter on /records endpoint

### PR DESCRIPTION
**Pull request by stanlyDoge**
https://github.com/strimzi/strimzi-kafka-bridge/issues/171

### CODE REVIEW

File: src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java [hunk 1]
```diff
@@ -126,10 +126,18 @@ public void handle(Endpoint<?> endpoint) {
...
+                if (routingContext.request().getParam("max_bytes") != null) {
+                    this.maxBytes = Long.parseLong(routingContext.request().getParam("max_bytes"));
+                }
+
                 this.consume(records -> {
                     if (records.succeeded()) {
                         Buffer buffer = (Buffer) messageConverter.toMessages(records.result());
-                        sendConsumerRecordsResponse(routingContext.response(), buffer);
+                        if (buffer.getBytes().length > this.maxBytes) {
+                            routingContext.response().setStatusMessage("Response is too large").setStatusCode(ErrorCodeEnum.UNPROCESSABLE_ENTITY.getValue()).end();
```
Comment (ppatierno) [inline src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java]: We should try to put the sending logic in the sendConsumerRecordsResponse refactoring it if needed with more parameters. I noticed that we should do that for other use cases.

Comment (stanlyDoge) [reply src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java]: But that is not matter of this PR, right?

**Comment by ppatierno** [comment]
Well in this case you need a way to reply to the client and it is just a little refactoring for using a method. But if you prefer we can do with another PR. Whatever you want :)

File: src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java (hunk 1 above)
Comment (stanlyDoge) [reply src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java]: Ok, I've done it. It was just a little change. I thought the refactoring would influence more of codebase.

**Pull request merged**